src/main/java/org/apache/mesos/hdfs/config/SchedulerConf.java

```diff
@@ -150,4 +168,35 @@ public String getHaZookeeperQuorum() {
     return getConf().get("mesos.hdfs.zkfc.ha.zookeeper.quorum", "localhost:2181");
   }
 
+  public String getStateZkServers() {
+    return getConf().get("mesos.hdfs.state.zk", "localhost:2181");
+  }
+
+  public int getStateZkTimeout() {
+    return getConf().getInt("mesos.hdfs.state.zk.timeout.ms", 20000);
+  }
+
+  public String getNativeLibrary() {
+    return getConf().get("mesos.native.library", "/usr/local/lib/libmesos.so");
+  }
+
+  public String getFrameworkHostAddress() {
+    String hostAddress = getConf().get("mesos.hdfs.framework.hostaddress");
+    if (hostAddress == null) {
+      try {
+        hostAddress = InetAddress.getLocalHost().getHostAddress();
+      } catch (UnknownHostException e) {
+        throw new RuntimeException(e);
+      }
+    }
+    return hostAddress;
+  }
+
+  public int getConfigServerPort() {
+    String configServerPortString = System.getProperty("mesos.hdfs.config.server.port");
```

Comment:
Gotcha, in case it was launched by marathon. Could you add a comment to make sure people aren't confused why we are checking the environment variables and a config file and don't get confused? :)